pages/project-version/routes.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
     path: '/pdf',
     router: pdf
   },
+  ntsPdf: {
+    path: '/pdf/nts',
+    router: pdf
+  },
```

Comment:
I think this is completely redundant on the basis that `/pdf/nts` will have already been matched to `/pdf` and routed to the `pdf` router in the block above.

It's not doing any harm though.

Reply:
It's useful for the link here: https://github.com/UKHomeOffice/asl-pages/pull/770/files#diff-e44a7db1107d243b630f6e677a105d10R54 rather than passing a suffix.

Reply:
Makes sense. 👍